core/transactions-factories/transfer_transactions_factory.md

```diff
@@ -12,6 +12,7 @@ class TransferTransactionsFactory:
         sender: IAddress;
         receiver: IAddress;
         native_amount: Amount;
+        data: Optional[bytes];
```

Comment:
In the specs, we usually say: `data?: bytes`.

Reply:
will keep in mind to change in the next PR. Thank you!